docs/portal/transports/manage-transport.md

```diff
@@ -7,9 +7,9 @@ To display this page, click **Transports** in the left pane.
 
 - To **filter transports**, type characters in the search box and then press the Enter key or click Magnifier icon. All transports that do not contain the typed characters will be hidden. Filtering is not case-sensitive. To remove the filter, click the "Reset" button.
 
-- To **view transport**, click the transport name allow you to view detailed configuration about the transport.
+- To **view transport**, click the transport name to display detailed configuration information about the transport.
 
-- To **Edit, Show History, Delete, Active/Suspend** a transport, click the "Action" dropdown box, and select the action you want to take.
+- To **Edit**, **Show History**, **Delete**, **Active**, or **Suspend** a transport, click the "Action" dropdown box, and select the action you want to take.
```

Comment:
Please hyphenate "dropdown" as follows: "drop-down."
Ref: https://sites.udel.edu/it-style/vocabulary-guidelines/#:~:text=drop%2Ddown%20is%20hyphenated%20when%20used%20as%20an%20adjective.&text=drop%2Ddown%20menu%20(aka%20drop,when%20used%20as%20an%20adjective.

Please consider changing "you want to take" with "you want to perform."